.github/workflows/deps.yaml

```diff
@@ -167,7 +179,121 @@ jobs:
         with:
           name: linux_arm64_substrait
           path: deps/linux_arm64
+      - uses: actions/download-artifact@v4
+        with:
+          name: iceberg-v1.1.3-extension-linux_arm64
+          path: deps/linux_arm64
       - name: Push static libraries
         uses: stefanzweifel/git-auto-commit-action@v5
         with:
           commit_message: Re-build static libraries
+
+  iceberg:
+    name: Build extension binaries
+    uses: ./.github/workflows/iceberg_extension_distribution.yaml
+    with:
+      extension_name: iceberg
+      duckdb_version: v1.1.3
+      ci_tools_version: v1.1.3
+      exclude_archs: 'linux_amd64_gcc4'
+
+  test_darwin_amd64:
```

Comment:
all 4 of htese jobs should use a matrix rather than repeating the code

Reply:
Done! I only have 2 jobs now. Couldn't make the tests run on arm64.